test/unit/style/style.test.js

```diff
@@ -1782,13 +1782,13 @@ test('Style#queryRenderedFeatures', (t) => {
         style._updateSources(transform);
 
         t.test('returns feature type', (t) => {
-            const results = style.queryRenderedFeatures([{column: 1, row: 1, zoom: 1}], {}, transform);
```

Comment:
First argument was supposed to be `{viewport, worldCoordinate}`, so apparently this was just working by accident before.